readme.md

```diff
@@ -55,9 +64,15 @@ It just so happens you will be learning one of the best ORM's on the market. It
 
 > Active Record is the M in MVC - the model - which is the layer of the system responsible for representing business data and logic. Active Record facilitates the creation and use of business objects whose data requires persistent storage to a database. It is an implementation of the Active Record pattern which itself is a description of an Object Relational Mapping system. - Taken from AR docs
 
-## Active Record
+## ActiveRecord
 
-For you to use Active Record to write ruby which manipulates data, we need to be able to talk about the **models** of our data.
+ActiveRecord is like a library that implements the Active record pattern in Ruby.
```

Comment:
This sounds like a word that uses itself in its own dictionary definition. They won't learn anything from this sentence.

Instead, a very high level sentence on how it allows us to interact with a database using Ruby would be more helpful.